onnxruntime/test/framework/inference_session_test.cc

```diff
@@ -330,6 +342,69 @@ TEST(InferenceSessionTests, DisableCPUArena) {
   RunModel(session_object, run_options);
 }
 
+TEST(InferenceSessionTests, TestModelSerialization) {
+  // Load model with level1 transform level as session options 
+  // and serialize the model after transformation.
+  SessionOptions so;
+  const string test_model = "testdata/transform/abs-id-max.onnx";
+  so.session_logid = "InferenceSessionTests.TestModelSerialization";
+  so.enable_cpu_mem_arena = false;
+  so.graph_optimization_level = TransformerLevel::Default;
```

Comment:
nits: 

don't think you need to set enable_cpu_mem_arena here or on line 381
line 357: Assert that model has Identity nodes
line 368: identify -> Identity
line 378: tranform -> transform